bgp_config_test.go

```diff
@@ -0,0 +1,53 @@
+package packngo
+
+import (
+	"testing"
+)
+
+var configID string
+var configRequest CreateBGPConfigRequest
+
+func TestAccCreateBGPConfig(t *testing.T) {
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, _ := setupWithProject(t)
```

Comment:
This compiles only because the projectID is declared in the package in another file. I would advice against using package variables, like configID, projectID, deviceID, or whatever you are likely to use in a function and expect it to be a local name.